cmd/main.go

```diff
@@ -62,7 +62,8 @@ func main() {
 	}
 
 	loader := manifestreader.NewManifestLoader(f)
-	invFactory := inventory.ClusterClientFactory{StatusPolicy: inventory.StatusPolicyNone}
+	// TODO: Add StatusPolicy back to factory.
```

Comment:
Does ConfigMapClientFactory need configurable StatusPolicy? Or is it configured elsewhere?

Reply:
ConfigMapClientFactory does need configurable StatusPolicy. We have historically hardcoded `StatusPolicyNone` event when we have a configurable field for `kapply`. Perhaps we should also add a flag to `kapply` to have users set this?

Regardless, ConfigMap now has a configurable StatusPolicy field for handling status conversion to the data fields.